Microsoft.Toolkit.Uwp.Services/Services/OneDrive/OneDriveService.cs

```diff
@@ -361,15 +361,18 @@ public bool Initialize(string appClientId, AccountProviderType accountProviderTy
                 }
             }
 
-            var requestMessage = Provider.Drive.Items[oneDriveId].Content.Request().GetHttpRequestMessage();
-            await Provider.AuthenticationProvider.AuthenticateRequestAsync(requestMessage).AsAsyncAction().AsTask(cancellationToken);
-            var downloader = completionGroup == null ? new BackgroundDownloader() : new BackgroundDownloader(completionGroup);
-            foreach (var item in requestMessage.Headers)
-            {
-                downloader.SetRequestHeader(item.Key, item.Value.First());
-            }
-
-            return await Task.Run(() => downloader.CreateDownload(requestMessage.RequestUri, destinationFile), cancellationToken);
+            return await Task.Run(
```

Comment:
@hermitdave I've moved the wohole code to Task.Run - what do you think about it? Do we need also configuring await here?